change/@fluentui-react-nav-preview-71eeaf96-dc68-4684-9b39-e17f06cdd4c1.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31704&commitId=f6e41b00164074aa518b8426409cec3b173098f8&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask - RTL.normal.chromium.png|2| Changed|
|Avatar Converged.basic - Dark Mode.normal.chromium.png|0| Added|
|Avatar Converged.basic - RTL.normal.chromium.png|0| Added|

</details>


  <details><summary><b>Card Converged - Interactive</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged - Interactive.appearance interactive - Filled - High Contrast.click.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Filled - High Contrast.hover.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Filled - High Contrast.normal.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Filled Alternative - Dark Mode.click.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Filled Alternative - Dark Mode.hover.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Filled Alternative - Dark Mode.normal.chromium.png|0| Added|

</details>


  <details><summary><b>Card Converged - Selectable</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged - Selectable.appearance selectable - Filled - High Contrast.click.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Filled - High Contrast.hover.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Filled - High Contrast.normal.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Filled - High Contrast.selected.chromium.png|0| Removed|

</details>


  <details><summary><b>SwatchPicker Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|SwatchPicker Converged.Spacing.default.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>